docs/api/storage/KVStoreAPI.md

```diff
@@ -0,0 +1,23 @@
+# Components implementing KVStore API
```

Comment:
Can you explain the difference between this API and the global here? When would someone use this one over that one?

Reply:
Also, can you explain what we mean by components and how to use them?

Reply:
Added explanation
Components-->classes